src/App.js

```diff
@@ -25,6 +25,10 @@ const App = () => {
 
   let currentLanguage = i18n.language;
 
+  useEffect(() => {
+    document.title = t("docTitle");
+  }, [currentLanguage, t]);
```

Comment:
Why is `t` a dependency as well? I might be missing something but it seems like you don't need it. If you do need it, then maybe you might not need `currentLanguage` too.

Reply:
Thanks for pointing out that each view should have its own title.

And, good question! I originally didn't have 't' as a dependency, but then received an es-lint warning: "React Hook useEffect has a missing dependency: 't'. Either include it or remove the dependency array." I tried taking 't' out and everything still seems to work. I only want the useEffect hook to run when the language changes, so it seems like taking it out should be okay